content/en/platform/corda/5.0-alpha/deploying/deployment-options/k8s-deployment.md

```diff
@@ -0,0 +1,14 @@
+---
+date: '2022-14-11'
+title: "Kubernetes Cluster"
+menu:
+  corda-5-alpha:
+    identifier: corda-5-alpha-deploy-local
+    parent: corda-5-alpha-deploy-options
+    weight: 2000
+section_menu: corda-5-alpha
+---
+Corda 5 Alpha is intended for deployment to a Kubernetes cluster using Amazon Elastic Kubernetes Service (EKS).
```

Comment:
Isn't it more proper Amazon=>AWS ?

Reply:
@davidcurrie can you confirm please?

Reply:
Although it surprises me, it is consistently self-styled as "Amazon EKS" on https://aws.amazon.com/eks/ so I vote to leave it as it is.